otx/algorithms/segmentation/adapters/mmseg/models/__init__.py

```diff
@@ -33,7 +33,7 @@
 __all__ = [
     "LiteHRNet",
     "MMOVBackbone",
-    "CustomFCNHead",
+    "get_head",
```

Comment:
Basically, `get_head()` function itself is useful to avoid duplicate implementation.

However, adding `get_head()` function in `__init__.py` and `heads/` looks inappropriate to me. 
It seems more pretty to locate `get_head()` function to `utils/` folder and implement `CustomOTXHead` in `heads/`, `__init__.py`.

Want to listen to other colleague's opinion. 
What do you think about this implementation? @supersoob , @eunwoosh

Reply:
I agree with @sungmanc. I hope we can deal with head init in CustomOTXHead.